exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/Otel2PrometheusConverter.java

```diff
@@ -642,4 +645,73 @@ private static String typeString(MetricSnapshot snapshot) {
     // Simple helper for a log message.
     return snapshot.getClass().getSimpleName().replace("Snapshot", "").toLowerCase(Locale.ENGLISH);
   }
+
+  private static String toLabelValue(AttributeType type, Object attributeValue) {
+    switch (type) {
+      case STRING:
+      case BOOLEAN:
+      case LONG:
+      case DOUBLE:
+        return attributeValue.toString();
+      case BOOLEAN_ARRAY:
+      case LONG_ARRAY:
+      case DOUBLE_ARRAY:
+      case STRING_ARRAY:
+        if (attributeValue instanceof List) {
+          return ((List<?>) attributeValue)
+              .stream()
+                  .map(String::valueOf)
+                  .map(it -> AttributeType.STRING_ARRAY.equals(type) ? toJsonValidStr(it) : it)
+                  .collect(Collectors.toList())
+                  .toString();
+        } else {
+          LOGGER.log(
```

Comment:
I think we need to assume that attributeValue is a List<Object>. If its not, throw an exception.